autogpt/config/config.py

```diff
@@ -59,7 +59,7 @@ def __init__(self) -> None:
             "PROMPT_SETTINGS_FILE", "prompt_settings.yaml"
         )
         self.fast_llm_model = os.getenv("FAST_LLM_MODEL", "gpt-3.5-turbo")
-        self.smart_llm_model = os.getenv("SMART_LLM_MODEL", "gpt-4")
+        self.smart_llm_model = os.getenv("SMART_LLM_MODEL", "gpt-3.5-turbo")
```

Comment:
This is a default change, but I think it's required. The vast majority of people don't have gpt-4 and we get complaints about it all the time.

Reply:
man you're so right, good catch, I completely forgot about that setting.